hugo/content/blog/2020-07-09-mumble-1-3-2-release.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Mumble 1.3.2 Release Announcement
+author: Robert Adam
+date: 2020-07-09
+categories:
+  - News
+  - Release
+  - Security
+---
+The Mumble team has released [**version 1.3.2**](https://github.com/mumble-voip/mumble/releases/tag/1.3.2) of the Mumble VoIP application. This is a
+**bugfix and security release** of the current and stable 1.3 series.

```

Comment:
Second reference to it being a security release.

Reply:
I thought this is how we call these minor releases. I'll change it though :point_up: